tests/wp-tests-bootstrap.php

```diff
@@ -55,7 +55,7 @@ static function () {
 tests_add_filter(
 	'muplugins_loaded',
 	static function () {
-		require dirname( dirname( __FILE__ ) ) . '/core-sitemaps.php';
+		require dirname( dirname( __FILE__ ) ) . '/sitemaps.php';
```

Comment:
Main plugin file is still `core-sitemaps.php`

Reply:
👍

Reply:
Fixed in https://github.com/GoogleChromeLabs/wp-sitemaps/pull/182/commits/76d8240f06999f1c7d2a2439ec4bf995204c3cef